typing/typecore.ml

```diff
@@ -369,12 +369,14 @@ let unify_exp_types loc env ty expected_ty =
       raise(Typetexp.Error(loc, env, Typetexp.Variant_tags (l1, l2)))
 
 (* helper functions for pattern environments *)
-let (!!) (penv : pattern_environment) = !(penv.env)
+let (!!) penv = Pattern_env.get_env penv
+(*
 let set_env (penv : pattern_environment) env = penv.env := env
 let get_equations_scope (penv : pattern_environment) =
   let l = penv.equations_scope in
   if l = lowest_level then invalid_arg "Typecore.get_equations_scope"
   else l
+*)
```

Comment:
I am not fond of commenting code out to disable it. It bitrots and becomes wrong, and people are not sure why it was left around and it stays forever. Could you just remove it?

Reply:
Oops, I just forgot to remove the scaffold. Done.